docs/archive_data_migration_to_berkeley.md

```diff
@@ -0,0 +1,470 @@
+# Migrating Mainnet Database to Berkeley
+
```

Comment:
I feel like we can add an overview, a "thanks for your patience" and "we have got you" type of introduction. The doc will benefit from a high level overview of the planning steps, execution steps, and verification steps.